app/views/widgets/_smart_banner.html.slim

```diff
@@ -0,0 +1,13 @@
+- cookie = cookies[:smart_banner_dismiss]
+- if (browser.device.mobile? || browser.device.tablet?) && (cookie.nil? || (Time.now >= Time.strptime(cookie, "%FT%R") + 15.days))
+  .smart-banner
+    .info
+      i.icon-close
+      a href="#{browser.platform.android? ? android_store_page : apple_store_page}"
+        = image_tag "icon", class: "app-icon"
+      h3 Baixe o app da Mudamos
+  javascript:
+    $(".smart-banner .info i.icon-close").click(function() {
+      $(".smart-banner").slideUp();
```

Comment:
I know it is pedantic, but you should be careful on situations like this one. If you happen to have more then one `.smart-banner` on the page, you would be closing all of them here. It is usually better to always scope your selectors:

```
$(".smart-banner .icon-close").on("click", function() {
  $(this).closest(".smart-banner").slideUp();
  ...
});
```